src/citrine/resources/data_concepts.py

```diff
@@ -40,8 +40,13 @@ class DataConcepts(PolymorphicSerializable['DataConcepts']):
 
     """
 
-    _local_keys = ['type']
-    """list of str: keys that appear in the serialized dictionary but not in the object itself."""
+    _type_key = "type"
+    """str: key used to determine type of serialized object."""
+
+    _client_keys = []
```

Comment:
1 possible interface would be to have use a dict here:
```python
_client_keys = {
  'audit_info': AuditInfo
}

def from_dict(cls, d):
  client_fields = []  
  for k, typ in _client_keys.items():
    vdict = d.pop(k)
    v = typ.from_dict(vdict)
    client_fields.append(v)
  ... # typical from_dict stuff
  for k, v in zip(_client_keys.keys(), client_fields):
    set_attr(obj, k, v)
```
This code is fake and won't run. Lmk if you need help understanding the approach I'm suggesting. Or if there's reason I'm overlooking to stick with something closer to what you've already written.

Reply:
Are you suggesting something like this go in `DictSerializable.from_dict()`? Or that `DataConcepts` get a `from_dict()` method?
The former doesn't make sense to me because if we're in `DictSerializable` then we in Taurus, below the level of the client.

Reply:
I _am_ suggesting that it goes in `DictSerializable`. Why would that be problematic? We will have the same issue in taurus if it interacts with the backend independent of citrine-python.

Reply:
It's not that it's problematic, it's just that if it's in `DictSerializable` then that suggests it's part of the data model, in which case we don't need to treat it differently.

Reply:
Yeah, my point here definitely assumes taurus is meant to see this field in one way or another.